src/app/api/auth/[...nextauth]/authOptions.ts

```diff
@@ -112,19 +113,33 @@ export const authOptions: NextAuthOptions = {
                 }
             },
         }),
+        KeycloakProvider({
+            clientId: `${process.env.SW360_KEYCLOAK_CLIENT_ID}`,
+            clientSecret: `${process.env.SW360_KEYCLOAK_CLIENT_SECRET}`,
+            issuer: `${process.env.AUTH_ISSUER}`,
+            checks: 'state',
+            authorization: { params: { scope: "openid READ WRITE" } },
+        }),
     ],
 
     session: {
         strategy: 'jwt',
     },
 
     callbacks: {
-        async jwt({ token, user }) {
+        async jwt({ token, user, account, profile }) {
+                //Initial Login
+                console.log("accunt", account)
+                console.log("profile", profile)
```

Comment:
No log on console.

Reply:
Had added it for debugging purposes. It's removed now.